courseraresearchexports/exports/api.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from courseraoauth2client import oauth2
+from courseraresearchexports.exports.constants import RESEARCH_EXPORTS_APP, \
+    RESEARCH_EXPORTS_API
+import requests
+
+
+def get(export_job_id):
+    """
+    Use Coursera's Research Export Resource to get a data export job given an
+    export job id.
+    :param export_job_id:
+    :return export_job:
```

Comment:
I think it makes sense to make `export_job` a Python class and add some behavior to this class. For example, things in utils such as `download_export`, and will allow us to have a safer API for `build_export_job_json`. 

It's good to encapsulate state and behavior within a class in OOP, to minimize the surface area of our code.